lib/reporters/base.js

```diff
@@ -118,9 +127,11 @@ exports.window = {
 };
 
 if (isatty) {
-  exports.window.width = process.stdout.getWindowSize
-    ? process.stdout.getWindowSize(1)[0]
-    : tty.getWindowSize()[1];
+  if (isBrowser) {
```

Comment:
would prefer stylistic stuff like this doesn't get changed, but NBD

Reply:
I ran into trouble with accessing `process.stdout.getWindowSize` when resolving browser/non-browser stuff in rollup with the previous code structure. Otherwise I would definitely have kept this block unchanged